src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/AuthorizationQueries.cs

```diff
@@ -24,9 +22,9 @@ public class AuthorizationQueries : IQuery<List<AuthOperation>, FindByUserIdCrit
         /// <param name="unitOfWork">Instance of class which implements <see cref="IUnitOfWork"/></param>
         public AuthorizationQueries(IUnitOfWork unitOfWork)
         {
+            VolleyUnitOfWork _unitOfWork;
```

Comment:
To move _unitOfWork to method where it is used

Reply:
Error code? Why it cannot be as private var?

Reply:
https://rules.sonarsource.com/csharp/RSPEC-1450

When the value of a private field is always assigned to in a class' methods before being read, then it is not being used to store class information. Therefore, it should become a local variable in the relevant methods to prevent any misunderstanding.

Reply:
The description Sonar provides is reasonable. But in this particular case this class works with UoW directly, other variables are more for convenience. Please revert changes related to _unitOfWork variable

Reply:
ok